obspy/taup/tau.py

```diff
@@ -303,6 +314,7 @@ def __init__(self, model="iasp91", verbose=False):
         """
         self.verbose = verbose
         self.model = TauModel.from_file(model)
+        self.flatteningOfEarth = earth_flattening
```

Comment:
I know the TauPy code is bad at this due to its Java origins, but please use snake case for new code.